src/api/services/lectureService.ts

```
@@ -0,0 +1,486 @@
+ */
+export const fetchLecturePlan = async (params: LecturePlanParams): Promise<Partial<LectureDetail>> => {
+  const { year, courseNumber } = params;
+  
+  // Validate courseNumber is 4 digits
+  if (!courseNumber || courseNumber.length !== 4) {
+    console.warn(`⚠️ Invalid courseNumber format: "${courseNumber}" (expected 4 digits)`);
+  }
+  
+  // Build KUPIS URL - ltShtm is fixed as B01012
+  const ltShtm = 'B01012';  // 고정값
+  const url = `https://kupis.konkuk.ac.kr/sugang/acd/cour/plan/CourLecturePlanInq.jsp?ltYy=${year}&ltShtm=${ltShtm}&sbjtId=${courseNumber}`;
```

ESLint

[error] 'url' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)